gateone/applications/terminal/plugins/ssh/scripts/ssh_connect.py

```diff
@@ -743,6 +751,7 @@ def main():
                     identities=parsed.get('identities', []),
                     additional_args=options.additional_args,
                     socket=options.socket,
+		    config=options.config,
```

Comment:
The indentation doesn't seem to be correct here. Try running `flake8` over this file.